packages/core/src/sync-realtime/service.ts

```diff
@@ -251,15 +252,16 @@ export class RealtimeSyncService extends Emittery<RealtimeSyncEvents> {
     } catch (error_) {
       if (this.isShuttingDown) return;
       const error = error_ as Error;
+      error.stack = undefined;
 
-      this.common.logger.warn({
+      this.common.logger.error({
         service: "realtime",
-        msg: `Realtime sync task failed (network=${
-          this.network.name
-        }, error=${`${error.name}: ${error.message}`})`,
+        error,
+        msg: `Realtime sync task failed (network=${this.network.name})`,
         network: this.network.name,
       });
-      // Note: Good spot to throw a fatal error.
+
+      this.emit("fatal", error);
```

Comment:
Are there any retries before this happens?

Reply:
Right now just the internal viem transport retries